### PR TITLE
FD-68 "Games are where the kids are" section

### PR DESCRIPTION
**Pull request by jwisc**

### CODE REVIEW

File: assets/scss/_home.scss [hunk 1]
```diff
@@ -125,6 +125,33 @@
...
         border-bottom: 4px solid #001E2A;
     }
+
+    .cabin {
+        position: absolute;
+        top: -2px;
+        width: max(903px, calc(100% + 5px));
+        height: 57vw;     
+        min-height: 400px;
+        max-height: 643px;
```
Comment (rwilkinson3) [inline assets/scss/_home.scss:135]: At the "Desktop - Huge" size the Figma mockup shows the cabin image extending vertically to the `<div class="bottom-frame"></div>` separator at the bottom of the level. I think this `max-height` is causing a gap, as the image doesn't reach the separator.

File: redesign.php [hunk 2]
```diff
@@ -25,7 +25,26 @@
             </div>
             <div class="hero-text">
                 <h4>Games let you try on a new identity.</h4>
-                <p>Games are an important tool for teachers because they allow players to take on the role of scientists, journalists, explorers, and more.</p>
+                <p>Games are an important tool for teachers because they allow players to take on the role of
+                    scientists, journalists, explorers, and more.</p>
+            </div>
+        </div>
+    </section>
```
Comment (rwilkinson3) [inline redesign.php:32]: Should add the `<div class="bottom-frame"></div>` separator between these levels.

**Review by rwilkinson3 — CHANGES_REQUESTED** [review]

**Review by rwilkinson3 — CHANGES_REQUESTED** [review]
I'm still seeing a gap between the cabin and bottom-frame divs as the cabin bobs up and down.
gap

Should add the `<div class="bottom-frame"></div>` separator between the helm and cabin levels.
bottom-frame

**Review by msherry2 — APPROVED** [review]
Thanks!

**Pull request merged**